test/coverage-test/coverage-report-tests/__snapshots__/istanbul.report.test.ts.snap

```diff
@@ -2518,7 +2518,7 @@ exports[`istanbul json report 1`] = `
       "0": {
         "loc": {
           "end": {
-            "column": null,
```

Comment:
I don't understand. How can a change to file filtering affect coverage columns? 🤔

Reply:
Not sure. This is the before-and-after:

- No visibile changes to end users
- No changes in coverage percentages

![image](https://github.com/vitest-dev/vitest/assets/14806298/f3391aea-0101-4d2c-b34e-dd261c8c7459)

Reply:
Looks like `testExclude.glob()` returns the files in different order now. Adding a simple `.sort()` restores the old behaviour.

Why branch endings are mapped differently by `istanbul-lib-coverage` or `istanbul-lib-source-maps` based on the file order - no idea. 🤷‍♂️